File: crypto/src/main/java/io/oversec/one/crypto/ui/util/ImageInfo.java
package io.oversec.one.crypto.ui.util;

public class ImageInfo {
    private final String mMimeType;
    private final int mWidth;
    private final int mHeight;

    public ImageInfo(String mimtype, int width, int height) {
        mMimeType = mimtype;
        mWidth = width;
        mHeight = height;
    }

    @Override
    public String toString() {
        return "ImageInfo{" +
                "mMimeType='" + mMimeType + '\'' +
                ", mWidth=" + mWidth +
                ", mHeight=" + mHeight +
                '}';
    }

    public int getWidth() {
        return mWidth;
    }

    public int getHeight() {
        return mHeight;
    }

    public String getMimetype() {
        return mMimeType;
    }
}

File: crypto/src/main/java/io/oversec/one/crypto/ui/util/ImageInfo.kt
package io.oversec.one.crypto.ui.util

class ImageInfo(val mimetype: String, val width: Int, val height: Int) {

    override fun toString(): String {
        return "ImageInfo{" +
                "mMimeType='" + mimetype + '\''.toString() +
                ", mWidth=" + width +
                ", mHeight=" + height +
                '}'.toString()
    }
}
